tests/auxtel/test_latiss_acquire_and_take_sequence.py

```diff
@@ -0,0 +1,414 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["LatissAcquireAndTakeSequence"]
+
+import random
+import unittest
+import asynctest
+import logging
+import asyncio
+
+from lsst.ts import salobj
+from lsst.ts import standardscripts
+from lsst.ts import externalscripts
+from lsst.ts.externalscripts.auxtel import LatissAcquireAndTakeSequence
+import lsst.daf.persistence as dafPersist
+
+random.seed(47)  # for set_random_lsst_dds_domain
+
+logging.basicConfig(level=logging.DEBUG)

```

Comment:
Do you really need to do this logging stuff? you can get the logging info from the unit test running 

```
pytest -vs --log-level DEBUG
```

Reply:
i got rid of the log level issue, but the matplotlib stuff is somewhat required or it's impossible to diagnose anything